portal/templates/service-options/service-options.njk

```diff
@@ -0,0 +1,24 @@
+{% extends "service-options/index.njk" %}
+
+{% block pageTitle %}Service Options{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl" data-cy="main-heading">What do you want to do?</h1>
```

Comment:
I know we have E2E tests that do the same thing but can we also add a couple of quick component tests for rendering this page with the canAccess variables as well please?